l10n_ro_account_edi_ubl/models/l10n_ro_account_anaf_sync_scope.py

```diff
@@ -12,12 +12,15 @@ class AccountANAFSyncScope(models.Model):
 
     scope = fields.Selection(selection_add=[("e-factura", "E-factura")])
 
-    @api.onchange("scope")
```

Comment:
salut, eu am lasat asta ca sa nu completeze utilizatorul manual linkurile...cand pui efactura sa fie completate automat, cand il schimbi in production sau test sa recalculeze...poate e mai usor sa depinda compute de anaf_sync_test_url si anaf_sync_production_url

Reply:
nu trebuie sa completeze utilizatorul linkuri si nu trebuie sa fie stocate in baza de date

Reply:
trebuie ținut cont și de scriptul de neutralizare

Reply:
Pai nu e mai usor...decat sa facem deploy de fiecare data cand schimba anaf ceva...sau poate vrem sa facem o intermediere cu inca o validare si folosesti alte url-uri decat cele de pe ANAF?

Reply:
inainte de acest fix era o combinatie dintre onchange si compute care nu functiona

Reply:
URL trebuie sa fie editabile de user, poate pe group_no_one. Intentia mea era sa avem un singur field apelabil cu URl computed in functie de status.

Reply:
de ce sa fie url editabil ?

Reply:
cand schimba anaf sa nu fie nevoie de un PR special pentru a modifica URL. De ce sa schimbe anaf? Nu stiu e bine sa luam in calcul situatia posibila.

Reply:
pe branch 16.0 am reusit sa obtin o linie cu Production si fara anaf_sync_url
![2024-03-15_09-41](https://github.com/OCA/l10n-romania/assets/10978901/b266ee66-6e5e-47da-abf2-3d7246ef4d81)